db/migrate/20231024213216_create_users.rb

```diff
@@ -10,3 +10,4 @@ def change
     end
   end
 end
+User.create(name: 'Jonh Doe', photo: 'https://plus.unsplash.com/premium_photo-1690481529194-6087914e096e?auto=format&fit=crop&q=80&w=1974&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D', bio: 'Teacher from Mexico',post_counter: 0)
```

Comment:
- [ ] Nice work adding a code line to create a user. However, this code can't be executed in the migration file simply because while migrating Rails walks through each migration file to first form the database, and it can't understand how to create a user while doing this (without a database to store that user in!). The right place for this code is the `seeds.rb` file that's being run by the command line `rails db:seed`. 🙏